lib/discourse_theme/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module DiscourseTheme
-  VERSION = "0.7.6"
+  VERSION = "0.7.7"
```

Comment:
```suggestion
  VERSION = "0.8.0"
```

I believer most gems follow [semver](https://semver.org/) which means we should be bumping the minor version when adding new features in a backwards compatible manner.